packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/interactive-dark-theme" as *;
+
+// ----------------
+// Q: Create tokens?
+$app-header-home-link-padding: 4px;
+
+.hds-app-header {
+  display: flex;
+  gap: 16px;
+  align-items: center;
+  justify-content: space-between;
+  min-height: 52px; 
+  padding: 8px 16px;
+  color: #FFF;
+  background: var(--token-color-palette-neutral-700);
+  border-bottom: 1px solid var(--token-color-palette-neutral-500);
+  box-shadow: 
+    0 4px 4px -2px hsla(0deg, 0%, 0%, 20%),
+    0 2px 3px 0 hsla(0deg, 0%, 0%, 10%);
+
+  // Apply dark theme to child interactive components
+  .hds-button,
+  .hds-dropdown-toggle-button,
+  .hds-dropdown-toggle-icon {
+    @include hds-interactive-dark-theme();
+  }
```

Comment:
@didoo Let me know if you think it's a bad idea to apply styles to the buttons & dropdowns by nesting this way...

Reply:
no, actually is quite neat 👍